spacecmd/src/spacecmd/utils.py

```diff
@@ -434,9 +434,10 @@ def print_errata_summary(erratum):
     if len(date_parts) > 1:
         erratum['date'] = date_parts[0]
 
-    print('%s  %s  %s' % (
+    print('%s  %s  %s  %s' % (
         erratum.get('advisory_name').ljust(14),
-        wrap(erratum.get('advisory_synopsis'), 50)[0].ljust(50),
+        wrap(erratum.get('advisory_synopsis'), 49)[0].ljust(49),
+        (erratum.get('advisory_status') or '').ljust(9),
```

Comment:
Instead of checking if `erratum.get` returns `None` and then setting the default value, it can be passed to `dict.get()` as the second parameter like this
```suggestion
        erratum.get('advisory_status', '').ljust(9),
```

I see the same is done further up, is this something that wasn't in Python 2 or is there another reason we've been using `dict.get() or X` until now?

Reply:
The reason is possible behavior in case the key exists in the dictionary, but the value is `null`: the `dict.get('key', '') would return `null` in such case.

Reply:
Thank you for the explanation, I didn't think about that case. I was expecting that `erratum.get()` would return `None` because the key might not be in the `dict`, not because the value is `None`. When the key is always in the dict, I favor `dict[key]` (i.e. `erratum['advisory_status']`) which might explain why I didn't not understand this immediately.